examples/huggingface/pytorch/code-generation/quantization/requirements.txt

```diff
@@ -5,6 +5,7 @@ protobuf
 sentencepiece != 0.1.92
 --extra-index-url https://download.pytorch.org/whl/cpu
 torch==2.1.0+cpu
+peft==0.7.0
```

Comment:
why quantization needs peft?

Reply:
https://github.com/intel/intel-extension-for-transformers/blob/main/intel_extension_for_transformers/llm/quantization/nn/modules.py#L23
woq needs call QBitsLinear class, but the peft is in the import list.

Reply:
remove it first and will have another PR to import the peft import only need for lora relatives by @XinyuYe-Intel .